handbook/engineering/web/goals.md

```diff
@@ -1,47 +1,119 @@
 
-### Medium term
+### _Frontend Platform_ - make the web app UI consistent, accessible, and scalable 
 
-_These medium-term goals are listed in order of rough priority. This means we preference a sooner-listed goal when making progress on one would conflict with another goal, but it doesn't mean we only work on the first goal until it's done – we balance our iterations in sum so we can make some progress on all of these goals over a quarter. Within each goal, the sub-goals are also listed in (stricter) priority order._ 
+**Problem:** 
 
-1. **Make code insights an entirely new reason to use Sourcegraph.**
-   1. Focus on building prototypes to solve the most pressing needs of our customers, to clearly demonstrate the value of code insights. 
-   1. Expose more generalizable metrics that let our users measure and track their own goals, whether those are migrations, code smells, security needs, cross-collaboration, or other information about code. 
-   1. Expose features that let our users build their own insights. 
+The Sourcegraph web platform has accumulated lots of design debt, since much of it was built before we had a design team. Additionally, the existing UI components don't scale to Sourcegraph's feature growth. The web platform should be more accessible, delightful, customizable, and scalable in order to empower Sourcegraph's users and Sourcegraph's internal teams' development.
 
-1. **Make Sourcegraph extensions a core part of Sourcegraph users' experiences.**
-   1. Track anonymized, general usage of extensions to determine which extensions are most successful at adding value for our users to inform our future work.
-   1. Build, maintain, and update API endpoints that are robust and immediately useful, to grow adoption of extensions. 
-   1. Make the extensions action bar clear and scalable, and help users discover and use our extensions. 
-   1. Improve discoverability and design of the extension registry to enable users to find the most useful extensions and make users excited to build their own extension. 
+### Milestones: 
+1. 🔄 Sourcegraph's core navigation effectively highlights our core features and scales to new ones. 
+   - **Outcome:** We have a defined path forward for expanding navigation to include new features without needing to frequently redesign. 
 
-1. **Increase the weekly active users of all our code host integrations.**
-   1. Maintain the existing native integrations and browser extensions. 
-   1. Build support for new code hosts and new browsers, like Safari and Gerrit. 
-   1. Build support for the browser extension to reference both a private Sourcegraph instance and public code on Sourcegraph.com. 
+1. Sourcegraph's UI lets users customize themes, colors, and font attributes. 
+   - **Outcome:**  The Sourcegraph UI meets all WCAG accessibility standards. 
```

Comment:
This seems more as an implementation detail to me than a goal, with the actual goal being something like _the Sourcegraph UI is accessible to all people, including those with different needs and/or preferences for themes, color contrast, fonts and font sizes_. For some aspects I think customizability would just be a cheap cop-out as opposed to making sure we play well with the OS's or browser's native accessibility settings.

Meeting WCAG accessibility standards seems a bit orthogonal to that, because there is the question of how do our _default_ colors/fonts etc hold up to these standards, and then how _customizable_ are those (obviously we can't guarantee standard compliance for custom themes).